src/plays/age-calculator-by-react/useAge.jsx

```diff
@@ -0,0 +1,93 @@
+import { useState } from 'react';
+
+const useAge = () => {
+  const [message, setMessage] = useState('');
+  const [year, setYear] = useState('-');
+  const [month, setMonth] = useState('-');
+  const [day, setDay] = useState('-');
+  const [birthday, setBirthday] = useState('');
+
+  const reset = (e) => {
+    e.preventDefault();
+    setMessage('Input your birthday');
+    setYear('-');
+    setMonth('-');
+    setDay('-');
+    setBirthday('');
+  };
+
+  const change = (e) => {
```

Comment:
Update variable names to be more descriptive. For example,` e` can be changed to `event`, and `months` can be renamed to `daysInMonth` to improve clarity.